backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/TypeImpl.java

```diff
@@ -205,13 +217,14 @@ public EList<FeatureMembership> getFeatureMembership() {
      */
     @Override
     public EList<Feature> getInheritedFeature() {
-        List<Feature> inheritedFeatures = new ArrayList<>();
-        this.getInheritedMembership().stream()
+        Feature[] data = getInheritedMembership().stream()
```

Comment:
Could you rename this variable from `data` to `inheritedFeatures` please?
Please rename every derived method variable you have modified from `data` to something more meaningful.

Reply:
Done